### DISCUSSION TITLE
How to show/integrate content of azure devops on wiki.js

### DISCUSSION THREAD

**Discussion by shweta952** [requarks/wiki#6716] [Help / Questions] [answered]
We want to show content of azure devops on wiki.js site. please suggest solution to integrate this.
How we can show the azure devops data on wiki.js. is it possible?

**Comment by NGPixel** [comment, marked as answer]
Use the git storage module.

**Reply by shweta952** [reply]
Thanks for input. But I need some inputs , how we can show azure devops details/contents on particular page of wiki.js
will it affect original wiki.js pages content.

**Comment by shweta952** [comment]
Thanks for input. But I need some inputs , how we can show azure devops details/contents on particular page  of wiki.js
will it affect original wiki.js pages content.
